jenkins-integration-build.sh

```diff
@@ -1,179 +1,164 @@
 #!/usr/bin/env bash
-
-# Uncomment these if running locally
-# export COUCHBASE_SERVER_ADDR=127.0.0.1
-# export WORKSPACE=.
-# export TARGET_PACKAGE="..."
-# export RUN_COUNT=1
-# export XATTRS=true
-# export GSI=false
-# export TARGET_TEST=ALL
-# export SG_EDITION=EE
-# export COUCHBASE_SERVER_PROTOCOL=couchbase
-# export TLS_SKIP_VERIFY=true
-# export SG_TEST_BUCKET_POOL_SIZE=3
-# export PACKAGE_TIMEOUT=120m
-# export DETECT_RACES=false
-# export TEST_DEBUG=true
-# export SG_TEST_PROFILE_FREQUENCY=""
-
-# Abort on errors
-set -e
-
-# Output all executed shell commands
-set -x
-
-# Check that Couchbase Server is reachable before we actually start
-curl --fail http://Administrator:password@$COUCHBASE_SERVER_ADDR:8091/pools/default
-
-# ANSI color sequences break XML report parsing, so can't enable
-#export SG_COLOR="true"
-
-# forces go to get private modules via ssh
-git config --global url."git@github.com:".insteadOf "https://github.com/"
-
-
-
-# Set up Go paths and check install
-export GO111MODULE=auto
-
-if [ -z $(echo :$PATH: | grep ":$(go env GOPATH)/bin:") ]; then
-  export PATH=$PATH:$(go env GOPATH)/bin
+DEFAULT_PACKAGE_TIMEOUT="45m"
+
+if [ $1 == "-m" ]; then
+    echo "Running in automated master integration mode"
+    # Set automated setting parameters
+    SG_COMMIT="master"
+    TARGET_PACKAGE="..."
+    TARGET_TEST="ALL"
+    RUN_WALRUS="true"
+    USE_GO_MODULES="true"
+    DETECT_RACES="false"
+    SG_EDITION="ee"
+    XATTRS="true"
+    RUN_COUNT="1"
+    # CBS server settings
+    COUCHBASE_SERVER_PROTOCOL="couchbase"
+    COUCHBASE_SERVER_VERSION="latest"
+    SG_TEST_BUCKET_POOL_SIZE="3"
+    GSI="false"
+    TLS_SKIP_VERIFY="false"
 fi
-echo "PATH: $PATH"
-
-go version
-go env
 
+set -e # Abort on errors
+set -x # Output all executed shell commands
+
+# Use Git SSH and define private repos
+git config --global --add url."git@github.com:".insteadOf "https://github.com/"
+export GOPRIVATE=github.com/couchbaselabs/go-fleecedelta
+
+# Use Go modules (3.1 and above) or bootstrap for legacy Sync Gateway versions (3.0 and below)
+if [ ${USE_GO_MODULES} == "false" ]; then
+  rm -rf *
+  wget https://raw.githubusercontent.com/couchbase/sync_gateway/$SG_COMMIT/bootstrap.sh
```

Comment:
Are these 2 steps really nessesary? They still basically prevent the script from being run locally in an existing sg repo, and AFAIK aren't required in Jeknins - because the job will already have a cloned copy of the repo at `$SG_COMMIT`